README.md

```diff
@@ -182,6 +182,18 @@ SELECT u.id FROM users as u
 
 will return `u.id` instead of just `id` if `columnsWithAlias=true`.
 
+##### `interpolateParams`
+
+```
+Type:           bool
+Valid Values:   true, false
+Default:        false
+```
+
+When `interpolateParams` is true, calls to `sql.Db.Query()` and `sql.Db.Exec()` with params interpolates placeholders (`?`) with given params. This reduces roundtrips to database compared with `interpolateParams=false` since it uses prepare, exec and close to support parameters.
+
+NOTE: It make SQL injection vulnerability when connection encoding is multibyte encoding except utf-8 (e.g. cp932).
```

Comment:
@arnehormann It's OK to check `mc.config.collation`.  But it's not perfect.
There is [ --skip-character-set-client-handshake](http://dev.mysql.com/doc/refman/5.0/en/server-options.html#option_mysqld_character-set-client-handshake) option.
And `SET NAMES cp932` can change connection charset online.